### PR TITLE
Implement design for login page

### PR DESCRIPTION
**Pull request by richtier**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 
login

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -418,3 +418,6 @@
 MADB_URL = env.str(
     'MADB_URL', 'https://www.check-duties-customs-exporting-goods.service.gov.uk'
 )
+
+# directory constants
+DIRECTORY_CONSTANTS_URL_SINGLE_SIGN_ON = env.str('DIRECTORY_CONSTANTS_URL_SINGLE_SIGN_ON', '')
```
Comment (sdonk) [inline config/settings.py]: Is this longer than 119? :)

File: core/context_processors.py [hunk 2]
```diff
@@ -10,5 +10,6 @@ def signup_modal(request):
         'linkedin_url': add_next(settings.SSO_OAUTH2_LINKEDIN_URL, url),
         'google_url':  add_next(settings.SSO_OAUTH2_GOOGLE_URL, url),
         'terms_url': urls.domestic.TERMS_AND_CONDITIONS,
+        'password_reset_url': urls.domestic.SINGLE_SIGN_ON / 'accounts/password/reset/',
```
Comment (sdonk) [inline core/context_processors.py]: Is this longer than 119? :)

File: config/settings.py (hunk 1 above)
Comment (richtier) [reply config/settings.py]: nope :) it's around 90

File: core/context_processors.py (hunk 2 above)
Comment (richtier) [reply core/context_processors.py]: nope :)

**Pull request merged**